monitoring/.codegen.yaml

```diff
@@ -1,5 +1,3 @@
 schemapatch:
   requiredFeatureSets:
   - "TechPreviewNoUpgrade"
```

Comment:
this isn't a default?

Reply:
No, at the moment, generating different feature sets is opt-in, had plan to revise that down the line but stuck with the behaviour that was set out when TP manifests were introduced